### PR TITLE
Example: validate

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
@mlastovski here is your task, please accept invitation to github organization

File: Tasks/validate.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Validate person name
+
+const isValidate = (T) => {
```
Comment (mlastovski) [inline Tasks/validate.js:3]: Not the greatest name of the function, should be changed to isValid. Unreadable parameter name, should be also changed, for instance, to 'username'

File: Tasks/validate.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+if (!T) return false;
```
Comment (mlastovski) [inline Tasks/validate.js:4]: Wrong tab spacing. Lots of 'if' conditionals, I would rewrite it to 1 line using logic operator OR (||)

File: Tasks/validate.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+if (T === '') return false;
+if (typeof T !== 'string') return false;
+if (T.length === 0) return false;
+if (!T.includes(' ')) return false; {
```
Comment (mlastovski) [inline Tasks/validate.js:8]: '{' to the next line, wrong spacing

File: Tasks/validate.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  for (C of T) {
+    if (C === ' ') continue;
+    if (C.toLowerCase().charCodeAt(0) >= 97 && C.toLowerCase().charCodeAt(0) <= 122) { } else { return false; }
```
Comment (mlastovski) [inline Tasks/validate.js:11]: Wrong styling. Here (line 11) I would change the logic, as here code is doing 1 unnecessary thing. 

            if (char.toLowerCase().charCodeAt(0) <= 97 && char.toLowerCase().charCodeAt(0) >= 122) {
                return false;
            }

**Comment by mlastovski** [comment]
```js
// Validate person name

const isValid = (username) => {
  if (
    !username ||
    username === "" ||
    typeof username !== "string" ||
    username.length === 0 ||
    !username.includes(" ")
  ) {
    return false;
  }

  {
    for (char of username) {
      if (char === " ") {
        continue;
      }

      if (
        char.toLowerCase().charCodeAt(0) <= 97 &&
        char.toLowerCase().charCodeAt(0) >= 122
      ) {
        return false;
      }
    }

    return true;
  }
};

const result = isValid("Mark Last");
console.log(result);

```

**Comment by mlastovski** [comment]
```py
# Validate person name

def is_valid(username):
    if not username or username == '' or type(username) is not str or ' ' not in username:
        return False
    
    for char in username:
        if char == ' ':
            continue

        if ord(char.lower()) <= 97 and ord(char.lower()) >= 122:
            return False

    return True
    

result = is_valid('Mark Last')
print(result)

```

**Pull request closed without merging**